pom.xml

```
@@ -28,7 +28,7 @@
 
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>plugin</artifactId>
-  <version>3.52-SNAPSHOT</version>
+  <version>4.0-SNAPSHOT</version>
```

```suggestion
  <version>3.52-SNAPSHOT</version>
```

Major bump is not really needed in the current implementation. The change is compatible AFAICT

---

Agreed, it 'should' be compatible. I'll need to make a quick docs. update too, because ISTR that referenced 4.0 ("can use this feature from...")

---

Updated docs to match 3.52: https://github.com/jenkins-infra/jenkins.io/pull/2634